zschema/compounds.py

```diff
@@ -58,12 +58,21 @@ def docs_es(self, parent_category=None):
             retv["doc"] = self.doc
         return retv
 
-    def validate(self, name, value):
+    def validate(self, name, value, policy=_NO_ARG, parent_policy=_NO_ARG):
+        calculated_policy = self._calculate_policy(name, policy, parent_policy)
         if type(value) != list:
-            raise DataValidationException("%s: %s is not a list",
-                                          name, str(value))
+            try:
+                raise DataValidationException("%s: %s is not a list",
```

Comment:
Should be `DataValidationException(fmt % (...))`, not `DataValidationException(fmt, ...)`, right?